packages/core/utils/lib/convert-query-params.js

```diff
@@ -215,6 +215,10 @@ const convertNestedPopulate = (subPopulate, schema) => {
     return true;
   }
 
+  if (_.isString(subPopulate)) {
+    return parseType({ type: 'boolean', value: subPopulate, forceCast: true });
```

Comment:
It allow extra values, it does `return Boolean(value)`. So it will be equivalent to true. I've added this option in order to avoid 500 errors because it's a regular error that is thrown. We can try catch somewhere to transform the error but I didn't think it was worth it. wdyt?

`false` will normally not populate the field (the function already can return false line 222). I remember allowing having false to not populate but I'm not sure if it was merged in main or not. I'll add a test :)

Reply:
Indeed `populate: false` was not supported in main. I added it in the PR with a test!